go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/spiceai/gospice/v5
+module github.com/spiceai/gospice/v6
 
 go 1.20
```

Comment:
@ewgenius let's bump the Go version to 1.22 too.

Reply:
fixed ci for 1.22